### PR TITLE
Add new feature for user

### PR DESCRIPTION
**Pull request by Altafalinadaf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+.mvn/
```
Comment (sheikhasim) [inline .gitignore:34]: good

File: readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+## test
```
Comment (sheikhasim) [inline readme.md:1]: changes this

**Review by sheikhasim — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Altafalinadaf) [reply .gitignore:34]: done

File: readme.md (hunk 2 above)
Comment (Altafalinadaf) [reply readme.md:1]: done

**Review by sheikhasim — APPROVED** [review]

**Pull request merged**